Lecture-6/Task1/l6task1/CoffeeSort.java

```diff
@@ -0,0 +1,15 @@
+package l6task1;
+
+public enum CoffeeSort {
+    ARABICA("Arabica"), ROBUSTA("Robusta");
+
+    private String sort;
```

Comment:
Field is redundant, it doesn't provide any sugnificant utility. You should use `CoffeeSort.ARABICA.name()` when Strings are needed